Added OpenAL test class.

awesomebirdy923 opened the pull request.

awesomebirdy923 committed:
Added OpenAL test class.

awesomebirdy923 committed:
Fixed the build-path issue.

DaveDLeague reviewed (dismissed):
jars should load automatically

DaveDLeague reviewed (approved):
basic sound working

awesomebirdy923 committed:
Added 3D sound

awesomebirdy923 committed:
Had to commit this.

I had to commit this because I changed the value of a variable and it won't let me checkout unless there are no changes.
